### PR TITLE
Install Increase gem

### PR DESCRIPTION
**Pull request by garyhtou**
Refactor codebase to use [Increase gem](https://github.com/garyhtou/increase-ruby)

_tested!_

### CODE REVIEW

File: app/services/increase_service.rb [hunk 1]
```diff
@@ -2,63 +2,21 @@
 
 class IncreaseService
   module AccountIds
-    def increase_environment
-      if Rails.env.production?
-        :production
-      else
-        :sandbox
-      end
+    def self.fs_main
```
Comment (cjdenio) [inline app/services/increase_service.rb]: These could maybe be constants (`FS_MAIN = Rails.application.credentials.dig(...)`)

File: config/initializers/increase.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  # Your Increase API Key!
+  # Grab it from https://dashboard.increase.com/developers/api_keys
+  config.api_key = Rails.application.credentials.dig(:increase, increase_envrionment, :api_key)
+
+  # The base URL for Increase's API.
+  # You can use
+  # - :production (https://api.increase.com)
+  # - :sandbox (https://sandbox.increase.com)
+  # - or set an actual URL
+  config.base_url = increase_envrionment
```
Comment (cjdenio) [inline config/initializers/increase.rb]:
```suggestion
  increase_environment = Rails.env.production? ? :production : :sandbox

  # Your Increase API Key!
  # Grab it from https://dashboard.increase.com/developers/api_keys
  config.api_key = Rails.application.credentials.dig(:increase, increase_environment, :api_key)

  # The base URL for Increase's API.
  # You can use
  # - :production (https://api.increase.com)
  # - :sandbox (https://sandbox.increase.com)
  # - or set an actual URL
  config.base_url = increase_environment
```

**Review by cjdenio — APPROVED** [review]
This looks awesome, thanks! I just left a few comments

File: config/initializers/increase.rb (hunk 2 above)
Comment (garyhtou) [reply config/initializers/increase.rb]: thanks for catching that

File: app/services/increase_service.rb (hunk 1 above)
Comment (garyhtou) [reply app/services/increase_service.rb]: good idea!

**Pull request merged**